src/e2e/app/kamu-node-puppet/Cargo.toml

```diff
@@ -0,0 +1,46 @@
+[package]
+name = "kamu-node-puppet"
+description = "Controller for kamu-node process"
+version = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+authors = { workspace = true }
+readme = { workspace = true }
+license-file = { workspace = true }
+keywords = { workspace = true }
+include = { workspace = true }
+edition = { workspace = true }
+publish = { workspace = true }
+
+
+[lints]
+workspace = true
+
+
+[lib]
+doctest = false
+
+
+[features]
```

Comment:
It is common to explicitly specify `default = []`.

Under some conditions I used to get an error for missing, but I don't remember the details exactly anymore